src/main/java/org/snomed/snowstorm/core/data/services/ConceptService.java

```diff
@@ -99,6 +99,9 @@
 	@Autowired
 	private ConceptAttributeSortHelper conceptAttributeSortHelper;
 
+	@Autowired
```

Comment:
I have updated the doFind method in ConceptService to set the ConcreteValue of Relationships.